src/mod/endpoints/mod_sofia/mod_sofia.h

```diff
@@ -913,6 +914,7 @@ struct private_object {
 	char *user_via;
 	char *redirected;
 	sofia_cid_type_t cid_type;
+	sofia_cid_type_t force_cid_type;
```

Comment:
I don't see this being used in your code.

Reply:
right , removed